services/core-web/src/components/noticeOfWork/applications/permitGeneration/NOWPermitGeneration.js

```diff
@@ -200,6 +206,7 @@ export class NOWPermitGeneration extends Component {
       : amendment.regional_office;
     permitGenObject.now_tracking_number = noticeOfWork.now_tracking_number;
     permitGenObject.now_number = noticeOfWork.now_number;
+    // permitGenObject.application_last_updated_date = formatDate(noticeOfWork.last_updated_date);
```

Comment:
Is this intentional? Can the line be removed or a note added as to why it's commented out temporarily if not?

Reply:
oh good catch, meant to delete.

Reply:
removed!